### DISCUSSION TITLE
Customizations from Bing (Cohorts, Page Pathing, Saved Filters)

### DISCUSSION THREAD

**Discussion by mattitoo** [apache/superset#20094] [Ideas]
Hello,

a couple of weeks ago there was a great presentation by people from Microsoft Bing about their „Project Titan“ showing various advanced customizations they have done based on Superset (and Clickhouse). 
The video can be found here: https://preset.io/events/how-the-bing-team-heavily-customized-superset-for-their-internal-data/

I tried contacting the presenters about their plans to open-source their customizations or contribute them to Superset. I am sure they are really busy and have not had a chance to reply yet.

Since we have plans to start working on some of the features shown there ourselves (esp. what they call „cohorts“ and the „Page Pathing“ feature) I wanted to ask if there has been any other communication with the Bing team or if there anybody else has interest on working on those topics.

**Comment by MarcinZegar** [comment]
bump
